src/Disciplina/Crypto/Serialise.hs

```diff
@@ -53,3 +55,10 @@ instance {-# OVERLAPPABLE #-}
     HasAbstractSignature ss a where
     unsafeAbstractSign sk = unsafeAbstractSign sk . serialise
     unsafeAbstractVerify pk = unsafeAbstractVerify pk . serialise
+
+-- | Type alias for denoting raw bytes. Indended to be used with hashes
+-- and signatures, like in type `Hash Raw`, and not type-safe hashing and
+-- signing.
+-- TODO: probably it makes sense to make it a newtype, like in Cardano?
+-- Also, is it the most appropriate place for it?
+type Raw = LBS.ByteString
```

Comment:
Should we use `Raw` type in `mrHash` field in `MerkleSignature`?

Reply:
Yes, that would be nice